### ISSUE TITLE
Broken Totem Cooldown function

### ISSUE THREAD

**Issue by TheoRetisch1** [Zrips/CMI#3065]
**Settings**
Cooldown -> Use: true
No permission for cmi.command.toggletotem

**Expected result**
All players will see the bossbar when using a totem and the specified cooldown will be used.

**Actual result**
Nobody sees the bossbar and the cooldown does not count. 

**Workaround**
For some reason, the bossbar is set to false for all players, although they were never able to change its status. If the status is false, the bossbar is not shown and the timer does not count. However, if you give the players cmi.command.toggletotem permission and let them manually set the status to true, the feature works as expected.

**additional information**
```
Totem:
  # When this set to true, on players death totem will be used even if he is not holding it in hand
  RemoveFromInventory: true
  Cooldown:
    # When this set to true player can use totem only every X second's
    Use: true
    Time: 10
  Warmup:
    # When this set to true player can use totem to have X amount of second's, during which he can die and be resurected
    # Totem will be consumed durring activation and wont be returned even if resurection is not used during warmup time
    Use: false
    Time: 10
```
Paper 1.13.2 - 653
CMI 8.6.21.5

**Comment by ApYukiple** [comment]
I am the same.
Even if I set it to true in config, when I start the server, it is fixed to the default value of false and the cooldown is not applied.

MC: 1.15.2
CMI :8.6.24.1
Spigot : last build

**Comment by ApYukiple** [comment]
Although it is not a solution. CMI8.6.16.5.jar works fine.
There seems to be other people who do not work with the latest version as well.

**Comment by TheoRetisch1** [comment]
Fixed in version 8.6.26.3 :)
